### DISCUSSION TITLE
Welcome to Get Discussions!

### DISCUSSION THREAD

**Discussion by kean** [kean/Get#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mgrider** [comment]
Somewhat sad to see no responses in here. I'm going to post a topic, so I figured I should introduce myself here too.

Hi, I'm Martin, 👋 

I'm an iOS developer based in Minneapolis, MN, USA. I have been making iOS apps since ~2008, and mostly do contract work. I've been a contractor since 2012, over 10 years now, and partly that is so I have time to work on my own apps and games. You can find some of them listed at abstractpuzzle.com.

I am using `Get` (and `Pulse` and `CreateAPI`) for a large-ish client project that I've been working on (part-time) for about 7 or 8 months now. It's been great!

**Reply by kean** [reply]
Hi Martin. I'm glad you like it, and you are using `Get`, `Pulse`, and `CreateAPI` together – precisely how it was designed to be used 😊
